### PR TITLE
Markup order pages

### PR DESCRIPTION
**Pull request by zpawn**

### CODE REVIEW

File: src/components/ViewOrder/OrderItem/OrderItem.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    <Stack basis={`${basis}%`}>
+        <img src={img} alt={name} style={{ width: "100%", height: "auto" }}/>
+    </Stack>
+);
+
+const NamePrice: FC<Partial<OrderItemType>> = ({ name, price, mos }) => (
+    <Stack justify="space-between" style={{ marginBottom: "8px" }}>
+        <Stack vertical>
+            <P5>{name}</P5>
+            <P11 style={{ color: lightTheme.colors.grey80 }}>{mos} mos</P11>
```
Comment (AshleyDawson) [inline src/components/ViewOrder/OrderItem/OrderItem.tsx:25]: You must use the Deskpro theme hook available from the SDK. If there's a theme change and/or dark mode added in future, we should inherit from the wider Deskpro product:

```typescript
import { useDeskproAppTheme } from "@deskpro/app-sdk";

export const Example = () => {
  const { theme } = useDeskproAppTheme();

  // ...
};
```

Comment (AshleyDawson) [reply src/components/ViewOrder/OrderItem/OrderItem.tsx:25]: You'll need to do this in all components using `lightTheme` directly, then remove all `import { lightTheme } from "@deskpro/deskpro-ui";` imports.

**Review by AshleyDawson — CHANGES_REQUESTED** [review]
Remove direct use of `lightTheme` and replace with `useDeskproAppTheme()` hook

File: src/components/ViewOrder/OrderItem/OrderItem.tsx (hunk 1 above)
Comment (zpawn) [reply src/components/ViewOrder/OrderItem/OrderItem.tsx:25]: got it. I'll replace it  in all components

Comment (zpawn) [reply src/components/ViewOrder/OrderItem/OrderItem.tsx]: done

**Review by AshleyDawson — APPROVED** [review]

**Pull request merged**